Expo1900/Expo1900/Model/Exposition.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Exposition.swift
+//  Expo1900
+//
+//  Created by 이영우 on 2021/04/06.
+//
+
+import Foundation
+
+struct Exposition: Decodable {
+    let title: String
```

Comment:
해당 이니셜라이저는 "만약 json 파일에 json 객체에 항목이 하나라도 빠지면, json 객체 자체가 nil 이 되는 것을 방지하고 싶어!" 라는 생각으로 작성하였습니다!
기본 값을 주는 것도 시도해 보았으나, 위 상황을 처리해주지는 못하더라고요 ㅠ

Reply:
json 객체에 항목이 빠지게 될 경우 (Struct 구조와 json 파일의 구조가 다르게 된다면) 위 처럼 처리를 해주게 되면 catch 문으로 넘어가게 됩니다. 
`The data couldn’t be read because it is missing.` 이런 error description이 나타날거예요ㅎㅎ 어떻게 처리해주면 좋을까요?

`"만약 json 파일에 json 객체에 항목이 하나라도 빠지면, json 객체 자체가 nil 이 되는 것을 방지하고 싶어!" ` 라는 접근 방법은 매우 좋은것 같습니다 👍🏻

Reply:
그럴 경우에는 Error 프로토콜을 정의해서 해당 Error에 대한 처리를 해주면 될 것 같아요!!